### ISSUE TITLE
IntelliSense absolutely refuses to find header files under all circumstances

### ISSUE THREAD

**Issue by adrianghc** [microsoft/vscode-cpptools#1911]
I've been trying to use Visual Studio Code on Windows to write a compile a C application on Linux. To this end, I'm using your extension and Ubuntu with the Linux Subsystem. My c_cpp_properties.json file looks as follows:

```
{
    "configurations": [
        {
            "name": "WSL",
            "intelliSenseMode": "clang-x64",
            "includePath": [
                "${workspaceRoot}",
                "${localappdata}/Packages/CanonicalGroupLimited.UbuntuonWindows_79rhkp1fndgsc/LocalState/rootfs/usr/lib/gcc/x86_64-linux-gnu/5/include",
                "${localappdata}/Packages/CanonicalGroupLimited.UbuntuonWindows_79rhkp1fndgsc/LocalState/rootfs/usr/local/include",
                "${localappdata}/Packages/CanonicalGroupLimited.UbuntuonWindows_79rhkp1fndgsc/LocalState/rootfs/usr/lib/gcc/x86_64-linux-gnu/5/include-fixed",
                "${localappdata}/Packages/CanonicalGroupLimited.UbuntuonWindows_79rhkp1fndgsc/LocalState/rootfs/usr/include/x86_64-linux-gnu",
                "${localappdata}/Packages/CanonicalGroupLimited.UbuntuonWindows_79rhkp1fndgsc/LocalState/rootfs/usr/include"
            ],
            "defines": [
                "__linux__",
                "__x86_64__"
            ],
            "browse": {
                "path": [
                    "${localappdata}/Packages/CanonicalGroupLimited.UbuntuonWindows_79rhkp1fndgsc/LocalState/rootfs/usr/lib/gcc/x86_64-linux-gnu/5/include",
                    "${localappdata}/Packages/CanonicalGroupLimited.UbuntuonWindows_79rhkp1fndgsc/LocalState/rootfs/usr/local/include",
                    "${localappdata}/Packages/CanonicalGroupLimited.UbuntuonWindows_79rhkp1fndgsc/LocalState/rootfs/usr/lib/gcc/x86_64-linux-gnu/5/include-fixed",
                    "${localappdata}/Packages/CanonicalGroupLimited.UbuntuonWindows_79rhkp1fndgsc/LocalState/rootfs/usr/include/x86_64-linux-gnu",
                    "${localappdata}/Packages/CanonicalGroupLimited.UbuntuonWindows_79rhkp1fndgsc/LocalState/rootfs/usr/include/*"
                ],
                "limitSymbolsToIncludedHeaders": true,
                "databaseFilename": ""
            },
            "cStandard": "c11",
            "cppStandard": "c++17"
        }
    ],
    "version": 3
}
```

Even though stdio.h exists in 
> ${localappdata}/Packages/CanonicalGroupLimited.UbuntuonWindows_79rhkp1fndgsc/LocalState/rootfs/usr/include/

IntelliSense refuses to find it (and all the others). Nothing I have tried has helped, including:

1. Disabling all extensions apart from the C/C++ extension.
2. Deleting the %appdata%\Code folder.
3. Deleting the .vscode folder.
4. Uninstalling and re-installing Visual Studio Code.

Absolutely everything I've tried has failed, and frankly I'm getting desperate, the kind of desperation you feel when you've been sitting on your PC for hours trying to solve a problem, having tried every single idea you could come up with, researching with every single sensible search keyword and frankly trying everything, all in vain. I wouldn't be surprised if in the end it turned out to be something stupid but right now I just need your help because whatever the problem could be, I don't see it.

The crazy thing is that I asked my brother to try it on their PC with my c_cpp_properties.json file, and it worked for him.

Please help. I'm running version 0.16.1 of the C/C++ extension on VS Code 1.22.2 (x64) on Windows 10 version 1803 (build 17134.5).

**Comment by sean-mcmanus** [comment]
This appears to be https://github.com/Microsoft/vscode-cpptools/issues/1694 , caused by a WSL change on Windows 10 April Update. We plan to have a fix for our 0.17.0-insiders2 build later this week.

**Comment by adrianghc** [comment]
Well, it seems like even though I searched quite a bit, I missed the right issue. At least now I know that there is nothing wrong on my end. I may have sounded silly in my desperation yesterday but I had wasted several hours on this and it was getting late with a deadline approaching (thankfully I still have a bit of time, hopefully enough). Anyway, thank you very much for providing the answer to this problem,  and thank you of course for publishing a fix so soon. I'll be sure to get it.
